codeigniter/framework/2015-10-31-1.yaml

```diff
@@ -0,0 +1,8 @@
+title: XSS attack vector in Security Library method xss_clean()
+link: http://www.codeigniter.com/user_guide/changelog.html#version-3-0-3
+cve: ~
+branches:
+    master:
+        time:     2015-10-27 12:30:18
+        versions: [<3.0.3]
```

Comment:
We can't install `1.x` and `2.x` via Composer.

To be accurate, `2.2.6` is not affected by it. http://www.codeigniter.com/userguide2/changelog.html

I don't know about `1.x`, but probably affected.

And `1.x` and `2.x` reached EOL.

Do I need to add the lower version?